qst/graphviz/src/main/java/io/deephaven/graphviz/LabelBuilder.java

```diff
@@ -135,10 +157,155 @@ public void visit(WhereTable whereTable) {
     }
 
     @Override
-    public void visit(GroupByTable groupByTable) {
-        sb.append("groupBy(");
-        append(Strings::of, groupByTable.columns(), sb);
-        sb.append(')');
+    public void visit(SingleAggregationTable singleAggregationTable) {
```

Comment:
Maybe we should just print out the key? I imagine Immutables generates nice toString()s.